css/style.css

```diff
@@ -0,0 +1,66 @@
+
+body {
+    background-color: #fdfdfd;
+    align-items: center;
+    justify-content: center;
+    display: block;
+}
+
+#main {
+    margin: 30px 8px;
+    padding: 15px;
+    border-radius: 5px;
+    background: #eee;
+} 
+
+#title {
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    font-weight: bold;
+    font-size: xxx-large;
+}
+
+.sub-title {
+    display: flex;
+    justify-content: center;
+    font-style: italic;
+}
+
+#image {
+    max-width: 100%;
+    display: block;
+    height: auto;
+    margin: 0 auto;
+    max-height: 20%;
+}
+
+#img-caption {
+    display: flex;
+    justify-content: center;
+    padding-bottom: 40px;
+}
+
+#tribute-info {
+    display: flex;
+    justify-content: center;
+    font-size: x-large;
+    margin: 15px 10px 15px 10px;
+}
+
+.read-more {
+    display: flex;
+    justify-content: center;
+    margin-top: 30px;
+}
+
+#tribute-link {
+    margin-left: 5px;
+}
+
+
+ 
+
```

Comment:
Please Do not add blank lines or spaces without reason. There should be only one free line between your CSS classes

Reply:
- Good job being polite while giving your review 👏 
- Please don't forget to give reasons why you are suggesting a change 💯 , this will help the student understand better why he needs to make the changes. 😄 😄 
- Please, I'd suggest you try using emojis while giving your reviews, this will make you sound more friendly 😄 😄